x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/DeleteExpiredDataIT.java

```diff
@@ -290,7 +289,7 @@ private void testExpiredDeletion(Float customThrottle, int numUnusedState) throw
         }
 
         // Before we call the delete-expired-data action we need to make sure the unused state docs were indexed
-        assertThat(indexUnusedStateDocsResponse.get().status(), equalTo(RestStatus.OK));
+        assertFalse(indexUnusedStateDocsResponse.get().hasFailures());
```

Comment:
This assertion is stronger (the previous one was a no-op); it looks from the comment that this is what was intended, so hopefully there's no adverse effects.